Automattic/jetpack
# Social: Remove share as a social post toggle

This change removes the share as a social post toggle

## Proposed changes:

* Removed the toggle from the UI
* Removed the deprecated meta option
* Added a version to the meta so we can validate on the backend

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/410
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Sandbox D153127-code
* Create a post, and make sure in the editor you do not see the share as a social post toggle
* Share it without attaching media, it should work as expected.
* Create another post, or add an attached media ( make sure to save to update meta this case ), then publish it.
* You should see the media being uploaded

Scenarios: p1719314535002709/1719314312.704179-slack-C02JJ910CNL
